google-codelab/google_codelab.js

```diff
@@ -28,6 +28,9 @@ const soy = goog.require('goog.soy');
 /** @const {string} */
 const TITLE_ATTR = 'title';
 
+/** @const {string} */
+const CODELAB_TITLE_ATTR = 'codelab-title';
```

Comment:
Done

Reply:
ha turns out I can't use a true @deprecated because then I can't use it without adding a bunch of warning suppressing. I've added a note to the description instead.